specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -365,6 +372,27 @@
           "maximum": 86400
         }
       }
+    },
+    "TumblingWindowTriggerReference": {
+      "description": "Tumbling window trigger reference type.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Tumbling window trigger reference type.",
+          "type": "string",
+          "enum": [

```

Comment:
>enum [](start = 11, length = 4)

Also use [x-ms-enum](https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum)

Reply:
Using x-ms-enum is blocked by hvermis

Reply:
@mcardosos We don't want enum types generated for this, it is more of a hint to customer that this is the value. Here's another example where our swagger has the same structure: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/datafactory.json#L3024

Reply:
OKok